src/main/java/io/aiven/kafka/connect/opensearch/BulkProcessor.java

```diff
@@ -439,6 +470,15 @@ private void handleMalformedDoc(final BulkItemResponse bulkItemResponse) {
                                     + " records. Ignoring and will not index record. Error was {}",
                             batchId, batch.size(), bulkItemResponse.getFailureMessage());
                     break;
+                case REPORT:
+                    final String errorMessage =
+                            String.format("Encountered a version conflict when executing batch %s of %s"
+                                            + " records. Reporting to DLQ and will not index record."
```

Comment:
Same thought RE assuming a DLQ topic:
```suggestion
                                            + " records. Reporting this error to the Connect runtime and will not index record."
```